virttest/virsh.py

```diff
@@ -3856,6 +3856,65 @@ def nwfilter_edit(name, options="", **dargs):
     return command(cmd, **dargs)
 
 
+def nwfilter_binding_create(name, options="", **dargs):
```

Comment:
function naming convention: verb + n.

Reply:
the virsh command is
```
nwfilter-binding-create xmlfile
```
so to make it the same as virsh command. 
its: n+verb